src/AppBundle/Entity/Ticket.php

```diff
@@ -246,4 +254,31 @@ public function getSeriesNumber(): string
     {
         return $this->seriesNumber;
     }
+
+    /**
+     * Set customerOrder
+     *
+     * @param UserOrder $userOrder
+     *
+     * @return Ticket
+     */
+    public function setUserOrder(UserOrder $userOrder = null)
```

Comment:
just add method removeOrder() and do what you need todo. Remove default ```null``` valued from $userOrder parameter